### PR TITLE
Fix documentation for Jdbc connection from Database

### PR DESCRIPTION
**Pull request by Uxio0**
# Pull Request Checklist

* [ ] Have you read [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)?
* [ ] Have you read through the [contributor guidelines](https://www.playframework.com/contributing)?
* [ ] Have you signed the [Lightbend CLA](https://www.lightbend.com/contribute/cla)?
* [ ] Have you [squashed your commits](https://www.playframework.com/documentation/2.4.x/WorkingWithGit#Squashing-commits)?
* [ ] Have you added copyright headers to new files?
* [ ] Have you checked that both Scala and Java APIs are updated?
* [ ] Have you updated the documentation for both Scala and Java sections?
* [ ] Have you added tests for any changed functionality?

## Fixes

Fixes #xxxx

## Purpose

What does this PR do?

## Background Context

Why did you take this approach?

## References

Are there any relevant issues / PRs / mailing lists discussions?

### CODE REVIEW

File: documentation/manual/working/javaGuide/main/sql/code/JavaJdbcConnection.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2009-2016 Typesafe Inc. <http://www.typesafe.com>
```
Comment (marcospereira) [inline documentation/manual/working/javaGuide/main/sql/code/JavaJdbcConnection.java]: Use the new header, pointing to Lightbend:

```java
/*
 * Copyright (C) 2009-2016 Lightbend Inc. <https://www.lightbend.com>
 */
```

**Comment by Uxio0** [comment]
Fixed the header not pointing to Lightbend

**Comment by marcospereira** [comment]
Build failed:

https://travis-ci.org/playframework/playframework/jobs/116238511#L570

You can validate your changes locally by running the following commands from inside playframework project:

```bash
cd framework
./bin/testDocumentation
```
Also, please squash your commits.

**Comment by Uxio0** [comment]
Should be Ok now

**Pull request merged**